perf/galley-load-tests/mcp-client-scaling-stress-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Discussion whether shell scripts longer than 5 lines should be written aside, can you at least use long options instead of short one-letter abbreviations?